Sources/SKSwiftPMWorkspace/SwiftPMWorkspace.swift

```diff
@@ -209,9 +209,9 @@ extension SwiftPMWorkspace: SKCore.BuildSystem {
       // We can't determine build settings for non-file URIs.
       return nil
     }
-    guard let path = try? AbsolutePath(validating: url.path) else {
-      return nil
-    }
+    guard let path = (url.withUnsafeFileSystemRepresentation {
```

Comment:
I'm surprised this is only needed in one place - why don't we have the same issue for incoming URLs from the editor?

Is there a reason you preferred `withUnsafeFileSystemRepresentation` instead of  `standardizedURL.path`?

Reply:
Well, this was specifically for the test, I'm sure that I will need to do this a fair amount more as we discover the sites.

The reason for that is mostly because I did this before we had a chat, and I just failed to think about `standardizedURL.path`,  I'll want to change to that and test it, as I think it would be better to use.

Reply:
Actually, thinking more about this, I don't think that `standardizedURL.path` will do what we want.  That will give the path as a URI string rather than the FSR, which means that it cannot be used for the `AbsolutePath(validating:)` constructor.